packages/dcos-checks/build

```diff
@@ -10,3 +10,13 @@ cd /pkg/src/github.com/dcos/dcos-checks
 
 go install -v
 cp -r /pkg/bin/ "$PKG_PATH"
+
+# Create the service file for all roles 
+service="$PKG_PATH/dcos.target.wants/dcos-poststart-checks.service"
```

Comment:
nit: We should call this `dcos-checks-poststart`, so that it'll be grouped next to `dcos-checks-prestart` once that exists. It'll also make tab-completing the unit name a little easier.